lib/get.js

```diff
@@ -4,7 +4,7 @@ module.exports = Get
 
 function Get (db, key, opts, cb) {
   this._db = db
-  this._node = new Node({key, flags: (opts && opts.hidden) ? Node.Flags.HIDDEN : 0}, 0, null, db.hash)
+  this._node = new Node({key, flags: (opts && opts.hidden) ? Node.Flags.HIDDEN : 0}, 0, null, db.hash, opts.sep)
```

Comment:
Wouldnt it be easier just to load this from the db instance? It's not really configurable per node/iterator anyway, but more of a db wide thing.

Reply:
Then we don't need all the options updates

Reply:
Yes sounds good! I'll work on this later.